client/common/src/util/markdown/markdown.ts

```diff
@@ -125,7 +125,6 @@ export const renderMarkdown = (
                 // Support different images depending on media queries (e.g. color theme, reduced motion)
                 source: ['srcset', 'media'],
                 // Support SVGs for code insights.
```

Comment:
```suggestion
                // Support SVGs for code insights.
```
Removing the comment would be good too!

Reply:
We still support SVGs right? Just not inside `object` tags?

Reply:
Yeah, I think the comment was specifically calling out the need for the special-case of `object` tags (which was specifically for Code Insights).